src/controllers/crop_controller.py

```diff
@@ -114,3 +117,42 @@ def add_weed(self, loc):
 
             return weed
 
+    def move_cursor_and_snap_selected_to_cursor(self, x, y, z):
```

Comment:
Could you clarify where this method is being used? What is this method's intended function?

Reply:
### The purpose of this method is to move of the crop' the parent  object（in this case, standskeleton）, while its connected children will also move along with it without causing separation, because the parent object will be attached to the moving cursor, while maintaining its own offset.  this method will also cause the object's origin to remain at the bottom of the object.So this method will be used while  move the object . Additionally, when placing a real object blender file into the project, you can use 'get name' method to chose the specified object in order to move the object around

Reply:
The purpose of this method is to move of the crop' the parent object（in this case, standskeleton）, while its connected children will also move along with it without causing separation, because the parent object will be attached to the moving cursor, while maintaining its own offset. this method will also cause the object's origin to remain at the bottom of the object.So this method will be used while move the object . Additionally, when placing a real object blender file into the project, you can use 'get name' method to chose the specified object in order to move the object around

Reply:
Thank you for the clarification. However, as mentioned in my comment below, how exactly is the cursor being used when Blender is being used via the API and not through the app?

Reply:
If you mean that this will centre the crop models to the origin please attach a picture of your output as I'm unable to replicate it on my local machine.

Reply:
From what Donghang explained to me yesterday, it sounds like this method moves the virtual cursor to a crop with skeleton attached and selects everything related to the crop including all subclasses. You can then move the selected objects. This only works with the new skeleton blender file which is not in this commit.

Reply:
Thank you for the clarification @wifimug

I'll be looking forward to the new Blender file